pipeline.py

```diff
@@ -72,26 +85,60 @@ class TextProcessingPipeline:
     """
 
     def __init__(self, corpus_manager: CorpusManager):
-        pass
+        self.corpus_manager = corpus_manager
 
     def run(self):
         """
         Runs pipeline process scenario
         """
-        pass
+        articles = self.corpus_manager.get_articles().values()
+        tokens = []
+        for article in articles:
```

Comment:
why do you need an intermediate variable?

Reply:
removed